styles/index.css

```diff
@@ -13,3 +13,9 @@
   width: 40rem;
   height: 40rem;
 }
+
+.card-container {
+  max-width: 400rem;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 4px 16px 0 rgba(0, 0, 0, 0.2);
```

Comment:
You can use a variable here if you want.

Reply:
Thanks for letting me know, variable used